src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+function createPartnerString(sex, partnerName) {
+  if (sex === 'f') {
+    return `${partnerName} is my husband`;
+  }
+
+  return `${partnerName} is my wife`;
+}
```

Comment:
No need for separate function here, logic is not reused.
Better to just create variable inside the component.